src/server/export-controller.js

```diff
@@ -24,7 +24,7 @@ class ExportController {
       const schemaHash = await this.database.getSchemaHash();
 
       if (schemaHash !== importingSystemSchemaHash) {
-        throw new BadRequest(`Database schema mismatch between request (${schemaHash}) and host (${importingSystemSchemaHash})`);
+        throw new BadRequest(`Database schema mismatch between importing system (${importingSystemSchemaHash}) and exporting system (${schemaHash})`);
```

Comment:
That is very helpful!

Reply:
I presume you were also confused by this inversion here 😅

Reply:
Yep, it's hard to guess.